frontend/src/pages/Confirmation.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import {NavBar} from "../components/NavBar";
+import {ListItem} from "../components/ListItem";
+import {ListInfo} from "../components/ListInfo";
+import "../styles/Confirmation.css";
+
+class Donut {
+    name: string
+    image: string
+
+    constructor(name: string, image: string) {
+        this.name = name;
+        this.image = image;
+    }
+}
+
+// Temporary Menu
+const items: Array<Donut> = [
```

Comment:
Linter seems to have caught that this array is unused.

Reply:
My bad. I resolved it now and resolved some merge conflicts.